### DISCUSSION TITLE
The log size of APIM in elasticsearch

### DISCUSSION THREAD

**Discussion by lyr3051** [Axway-API-Management-Plus/apigateway-openlogging-elk#187] [Q&A] [answered]
Hi
Are all size of logs be smaller than before after using this solution?
How do I  prove it ?

**Comment by cwiechmann** [comment, marked as answer]
I'm not sure, about this question, but I would say: No, logs are not smaller. Logs are fed into Elasticsearch, but the original logs stay as they are before..
